easybuild/easyconfigs/g/GPAW/GPAW-19.8.1-intel-2019a-ASE-3.18.0-Python-3.7.2.eb

```diff
@@ -0,0 +1,49 @@
+easyblock = "PythonPackage"
+
+name = 'GPAW'
+version = '19.8.1'
+aseversion = '3.18.0'
+pythonsuffix = '-Python-%(pyver)s'
+versionsuffix = '-ASE-' + aseversion + pythonsuffix
```

Comment:
Same here

Reply:
I will also rename (and amend) this file.